content/en/docs/instrumentation/js/getting-started/nodejs.md

```diff
@@ -149,8 +158,7 @@ To find all autoinstrumentation modules, you can look at the
 
 The instrumentation setup and configuration must be run _before_ your
 application code. One tool commonly used for this task is the
-[`-r, --require module`](https://nodejs.org/api/cli.html#cli_r_require_module)
-flag.
+[--require](https://nodejs.org/api/cli.html#-r---require-module) flag.
 
 Create a file named `instrumentation.ts|js`, which will contain your
```

Comment:
same confusing syntax

Reply:
TY, I think we have that syntax in a few other documents as well, let me hunt them down